modules/core/arrow-core-extensions/src/main/kotlin/arrow/core/extensions/try.kt

```diff
@@ -12,9 +12,9 @@ fun <A> Try<A>.combine(SG: Semigroup<A>, b: Try<A>): Try<A> =
   when (this) {
     is Success<A> -> when (b) {
       is Success<A> -> Success(SG.run { value.combine(b.value) })
-      is Failure -> this
+      is Failure -> b
     }
-    is Failure -> b
+    is Failure -> this
```

Comment:
This should be kept the same way, I believe for Scala reasons.

Reply:
I setted as Scala does.
On scala if you try to combine a Failure and a success returns a Failure

Reply:
Also the test which test this behavior failed with the previous implementation

Reply:
There was a reason why we had it this way, maybe it was *not* to be like Scala.

Reply:
https://github.com/typelevel/cats/pull/1128#issue-73804552
https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/Apply.scala#L225

something to consider, I believe @AdrianRaFo changes to the `Try.semigroup` may be correct and that should be the behavior of the default instance.